### PR TITLE
Added proselint config

### PR DESCRIPTION
**Pull request by wargio**
This should fix proselint issues.

### CODE REVIEW

**Comment by XVilka** [comment]
```
./src/crackmes/ioli/ioli_0x00.md:46:4: misc.but No paragraph should start with a 'But'.

./src/crackmes/avatao/01-reverse4/main.md:54:69: misc.chatspeak 'btw:' is chatspeak. Write it out.

./src/crackmes/avatao/01-reverse4/radare2.md:8:52: consistency.spacing Inconsistent spacing after period (1 vs. 2 spaces).

./src/crackmes/avatao/01-reverse4/vmloop.md:21:70: misc.but No paragraph should start with a 'But'.

./src/crackmes/avatao/01-reverse4/vmloop.md:185:1: misc.chatspeak 'BTW,' is chatspeak. Write it out.

./src/crackmes/avatao/01-reverse4/vmloop.md:289:82: misc.but No paragraph should start with a 'But'.

./src/crackmes/avatao/01-reverse4/intro.md:11:14: consistency.spacing Inconsistent spacing after period (1 vs. 2 spaces).

./src/debugger/files.md:5:125: misc.but No paragraph should start with a 'But'.

./src/visual_mode/intro.md:3:95: consistency.spacing Inconsistent spacing after period (1 vs. 2 spaces).

./src/analysis/symbols.md:3:83: misc.but No paragraph should start with a 'But'.

./src/signatures/zignatures.md:62:32: misc.but No paragraph should start with a 'But'.

./src/plugins/intro.md:47:4: misc.but No paragraph should start with a 'But'.

./src/refcard/radare2_rc.tex:8:51: garner.redundancy.ras RAS syndrome. Use 'PDF' instead of 'PDF format.'.

./src/refcard/intro.md:166:79: misc.but No paragraph should start with a 'But'.

./src/refcard/intro.md:181:26: lexical_illusions.misc There's a lexical illusion here: a word is repeated.

./src/basic_commands/print_modes.md:230:213: misc.but No paragraph should start with a 'But'.

./src/search_bytes/basic_searches.md:39:1: misc.but No paragraph should start with a 'But'.

./src/tools/rahash2/intro.md:19:103: misc.but No paragraph should start with a 'But'.

./src/tools/rafind2/intro.md:63:4: misc.but No paragraph should start with a 'But'.
``

**Comment by wargio** [comment]
@XVilka yes i know. i was fixing this while i wanted to test that the config was used by the CI

File: src/plugins/intro.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ dL         # list debug plugins
 mL         # list fs plugins
 ph         # print support hash algoriths
 ```
-But also using the `?` value in the associated eval vars.
+Also using the `?` value in the associated eval vars.
```
Comment (wargio) [inline src/plugins/intro.md:48]: i think this is broken english.

File: src/tools/rahash2/intro.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ Note that rahash2 also permits to read from stdin in a stream, so you don't need
 
 When doing forensics, it is useful to compute partial checksums. The reason for that is because you may want to split a huge file into small portions that are easier to identify by contents or regions in the disk.
 
-This will spot the same hash for blocks containing the same contents. For example, if filled by zeros.
+This will spot the same hash for blocks containing the same contents. For example, if filled by zeros, 
 
-But also, it can be used to find which blocks have changed between more than one sample dump.
+It also can be used to find which blocks have changed between more than one sample dump.
```
Comment (wargio) [inline src/tools/rahash2/intro.md:21]: `it can also be used`

**Pull request merged**